ui/analyse/src/study/description.ts

```diff
@@ -22,7 +22,7 @@ export class DescriptionCtrl {
   }
 
   set(t: string | undefined) {
-    this.text = t ? t : undefined;
```

Comment:
warning, change of semantic in case of empty string

Reply:
why do we want to set an empty string there?

Reply:
it was just a note to myself that the code-golf I did to produce assets had a change of behaviour in case of empty string.
Previously it'd not do anything, whereas afterwards it would set it to empty.